### DISCUSSION TITLE
Versioning of crossdb...

### DISCUSSION THREAD

**Discussion by kolinfluence** [crossdb-org/crossdb#30] [Q&A]
sorry to disturb. it's great progress no doubt, everything's moving in the right direction

but possible to keep this below version 1.0 and make it 0.9.N for at least til next year June - July?

it's great software but definitely hope to see some production use hiccup before it is 100% version 1 for embedded use.

at least battle tested.

**Comment by jcwangxp** [comment]
It'll be 0.10.0 0.11.0 ..., 1.0.0 will be next year.
